modules/account-lib/src/utils/crypto.ts

```diff
@@ -150,3 +151,18 @@ export function isValidEd25519PublicKey(pub: string): boolean {
     return false;
   }
 }
+
+/**
+ * Whether the input is a valid BLS private key
+ *
+ * @param {string} prv A hexadecimal public key to validate
+ * @returns {boolean} Whether the input is a valid public key or not
+ */
+export function isValidBLSPrv(prv: Uint8Array): boolean {
```

Comment:
I know you're just following the pattern but seems that the `keyPair` classes should have these verification functions IMO

Reply:
No action required here, just a comment